.github/workflows/trunk.yml

```diff
@@ -172,10 +173,10 @@ jobs:
         source .ci/scripts/utils.sh
         install_executorch
 
-        install_arm
+        .ci/scripts/setup-arm_baremetal.sh
 
-        # Run arm unit tests
-        pytest -c /dev/null -v -n auto --cov=./ --cov-report=xml backends/arm/test
+        # Run arm pytest tests with Corstone3x0 FVP testing
+        .ci/scripts/test_arm_baremetal.sh test_pytest_ethosu_fvp
```

Comment:
I guess firing fvp a bunch of time is more time consuming compared to running a single large-ish model like mv2?

Reply:
Maybe, we have a patch to make it run FVP faster ongoing also.

Reply:
But the webare also adding more FVP testing. Lets see how it pans out a bit over time.